tidb-cloud/changefeed-sink-to-apache-kafka.md

```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
 
 If your Apache Kafka service is in a GCP VPC that has no internet access, take the following steps:
```

Comment:
```suggestion
If your Apache Kafka service is in a GCP VPC that has no Internet access, take the following steps:
```

Reply:
According to [Capitalization of Internet - Wikipedia](https://en.wikipedia.org/wiki/Capitalization_of_Internet), the spelling internet has become often used, as the word almost always refers to the global network; the generic sense of the word has become rare in non-technical writings. As a result, various [style manuals](https://en.wikipedia.org/wiki/Style_guide), including [The Chicago Manual of Style](https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style), the [Associated Press](https://en.wikipedia.org/wiki/Associated_Press)'s [AP Stylebook](https://en.wikipedia.org/wiki/AP_Stylebook), and the [AMA Manual of Style](https://en.wikipedia.org/wiki/AMA_Manual_of_Style), revised their formerly capitalized stylization of the word to lowercase internet in 2016.